medicines/pars-upload/src/auth/authPopup.js

```diff
@@ -2,11 +2,11 @@ import { UserAgentApplication } from 'msal'
 import { msalConfig, loginRequest } from './authConfig'
 
 export async function getAccount() {
+  msalConfig.auth.redirectUri = getCurrentHost(window.location.href)
```

Comment:
Would it make sense to put this bit in `authConfig.js` instead?

Reply:
I had it there initially  but it couldn't access `window`